ControlFlow/Program.cs

```diff
@@ -15,22 +15,51 @@ public static class ControlFlow
         // be validated).  Negative numbers should return "Invalid".
         public static string AreYouValid(int number)
         {
-            return default;
+            string output = "Invalid";
+            if (number > 1 && number < 10)
+            {
+                output = "Valid";
+                return output;
+
+            }
+            return output;
```

Comment:
Why are there two return output here?
```suggestion
            return output;
```